src/features/game/types/consumables.ts

```diff
@@ -33,163 +55,12 @@ export type ConsumableName =
-    experience: 24,
-    building: "Fire Pit",
-    cookingSeconds: 60 * 3,
-    ingredients: {
-      Pumpkin: new Decimal(10),
-    },
-    marketRate: 16,
-  },
-
-  "Bumpkin Broth": {
-    name: "Bumpkin Broth",
-    description: "A perfect broth for a cold day.",
-    experience: 96,
-    building: "Fire Pit",
-    cookingSeconds: 60 * 20,
-    ingredients: {
-      Carrot: new Decimal(10),
-      Cabbage: new Decimal(5),
-    },
-    marketRate: 64,
-  },
-
-  "Boiled Eggs": {
-    name: "Boiled Eggs",
-    description: "Boiled Eggs are great for breakfast",
-    experience: 90,
-    building: "Fire Pit",
-    cookingSeconds: 60 * 60,
-    ingredients: {
-      Egg: new Decimal(5),
-    },
-    marketRate: 160,
-  },
-
-  "Kale Stew": {
-    name: "Kale Stew",
-    description: "A perfect Bumpkin Booster",
-    building: "Fire Pit",
-    cookingSeconds: 60 * 60 * 2,
-    ingredients: {
-      Kale: new Decimal(10),
-    },
-    experience: 400,
-    marketRate: 400,
-  },
-
-  "Roast Veggies": {
-    name: "Roast Veggies",
-    description: "Even Goblins need to eat their veggies!",
-    experience: 170,
-    building: "Kitchen",
-    cookingSeconds: 60 * 60 * 2,
-    ingredients: {
-      Cauliflower: new Decimal(15),
-      Carrot: new Decimal(10),
-    },
-    marketRate: 240,
-  },
-
-  "Bumpkin Salad": {
-    name: "Bumpkin Salad",
-    description: "Gotta keep your Bumpkin healthy!",
-    experience: 290,
-    building: "Kitchen",
-    cookingSeconds: 60 * 60 * 3.5,
-    ingredients: {
-      Beetroot: new Decimal(20),
-      Parsnip: new Decimal(10),
-    },
-    marketRate: 400,
-  },
-
-  "Goblin's Treat": {
-    name: "Goblin's Treat",
-    description: "Goblins go crazy for this stuff!",
-    experience: 500,
-    building: "Kitchen",
-    cookingSeconds: 60 * 60 * 6,
-    ingredients: {
-      Pumpkin: new Decimal(10),
-      Radish: new Decimal(20),
-      Cabbage: new Decimal(10),
-    },
-    marketRate: 800,
-  },
-
-  "Cauliflower Burger": {
-    name: "Cauliflower Burger",
-    description: "Calling all cauliflower lovers!",
-    experience: 255,
-    building: "Kitchen",
-    cookingSeconds: 60 * 60 * 3,
-    ingredients: {
-      Cauliflower: new Decimal(15),
-      Wheat: new Decimal(5),
-    },
-    marketRate: 304,
-  },
-
-  Pancakes: {
-    name: "Pancakes",
-    description: "A great start to a Bumpkins day",
-    experience: 480,
-    building: "Kitchen",
-    cookingSeconds: 60 * 20,
-    ingredients: {
-      Wheat: new Decimal(5),
-      Honey: new Decimal(10),
-    },
-    marketRate: 10,
-  },
-
-  "Club Sandwich": {
-    name: "Club Sandwich",
-    description: "Filled with Carrots and Roasted Sunflower Seeds",
-    experience: 170,
-    building: "Kitchen",
-    cookingSeconds: 60 * 60 * 3,
-    ingredients: {
-      Sunflower: new Decimal(100),
-      Carrot: new Decimal(25),
-      Wheat: new Decimal(5),
-    },
-    marketRate: 184,
-  },
-
+export const CAKES: Record<CakeName, Consumable> = {
```

Comment:
Is there any relevance to `CAKES` anymore?

I would suggest calling these `BAKERY_CONSUMABLES` so it is clear the use case